### ISSUE TITLE
Metadata editor breaks when you click on an image

### ISSUE THREAD

**Issue by matthias-ronge** [kitodo/kitodo-production#3732]
A new process was started. It went through the "scan" task, images were inserted, "generate derivatives" (automatic), and now the "metadata" task takes place. You open the metadata editor from task page, and it looks like this:

![Metadata editor](https://user-images.githubusercontent.com/3040657/83743097-87924900-a65a-11ea-8419-ff7d9b24bd53.jpg)

You click on any of the images in the "unstructured media" section, and it dies with Stacktrace:

```
java.lang.NullPointerException
    at org.kitodo.production.forms.dataeditor.StructurePanel.updatePhysSelectionInLogTreeRecursive(StructurePanel.java:854)
    at org.kitodo.production.forms.dataeditor.StructurePanel.updatePhysSelectionInLogTreeRecursive(StructurePanel.java:862)
    at org.kitodo.production.forms.dataeditor.StructurePanel.updateLogicalNodeSelection(StructurePanel.java:773)
    at org.kitodo.production.forms.dataeditor.StructurePanel.updateNodeSelection(StructurePanel.java:728)
    at org.kitodo.production.forms.dataeditor.GalleryPanel.updateStructure(GalleryPanel.java:352)
    at org.kitodo.production.forms.dataeditor.GalleryPanel.select(GalleryPanel.java:800)
    at org.kitodo.production.forms.dataeditor.GalleryPanel.selectMedia(GalleryPanel.java:734)
    at org.kitodo.production.forms.dataeditor.GalleryPanel.select(GalleryPanel.java:761)
```

**Goal:** When you click it, the picture should be selected.

**Comment by solth** [comment]
I suspect this only happens with processes that have a parent process.
@matthias-ronge can you confirm or deny this?

**Comment by matthias-ronge** [comment]
Closed as duplicate of #3627
